splice_machine/src/test/java/com/splicemachine/derby/test/framework/SpliceWatcher.java

```diff
@@ -249,6 +249,7 @@ public ResultSet executeQuery(String sql, String userName, String password) thro
         while (rs.next()) {
             resultList.add((T) rs.getObject(1));
         }
+        rs.close();
```

Comment:
I think a try-with-resources would be better, in case we raise an exception for instance.

Reply:
Made the change.